dir.props

```diff
@@ -69,6 +69,8 @@
     <Platform>AnyCPU</Platform>
     <OutputType>Library</OutputType>
     <RunApiCompat>true</RunApiCompat>
+    <!-- Default to using the open public key -->
+    <AssemblyOriginatorKeyFile>$(ToolsDir)Open.snk</AssemblyOriginatorKeyFile>
```

Comment:
Yes, I think we do, e.g. in the [System.Net.Http.csproj](https://github.com/dotnet/standard/blob/ec132d7b0528ef840438821cb066a35317b4213d/platforms/xamarin.ios/System.Net.Http.csproj) and [System.Web.Services.csproj](https://github.com/dotnet/standard/blob/ec132d7b0528ef840438821cb066a35317b4213d/platforms/xamarin.ios/System.Web.Services.csproj) (those seem to be the only two).